### DISCUSSION TITLE
请问播放列表支持按视频文件的修改时间降序排列吗？

### DISCUSSION THREAD

**Discussion by tyf2018** [hooke007/mpv_PlayKit#462] [Q&A 问答]
请问播放列表支持按视频文件的修改时间降序排列吗？

**Comment by hooke007** [comment]
原始功能不支持，需要外部的排序脚本

**Comment by hooke007** [comment]
已存在第三方脚本支持

**Comment by tyf2018** [comment]
> 已存在第三方脚本支持

您的配置包以后会考虑加进来么？

**Reply by hooke007** [reply]
不会，只接受uosc功能支持

**Reply by unknownnumbers** [reply]
大佬您好!
请问, playlist 相关支持插件叫什么, 有链接吗? 我想尝试使用 MPV 以替换 PotPlayer, 但困扰于 playlist 的支援. 希望能有比较强大的插件处理 playlist 问题. 万分感谢!

**Reply by hooke007** [reply]
https://github.com/CogentRedTester/mpv-file-browser/discussions/90
